streams/src/test/java/org/apache/kafka/test/InternalMockProcessorContext.java

```diff
@@ -136,6 +136,7 @@ public InternalMockProcessorContext(final File stateDir,
             metrics,
             null,
             cache);
+        super.setCurrentNode(new ProcessorNode("TESTING_NODE"));
```

Comment:
This is to support the new thing I added to set the current node during Processor init.